feature/permissions/src/main/java/com/google/jetpackcamera/permissions/PermissionsViewModel.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.permissions
+
+import android.Manifest
+import androidx.compose.runtime.mutableStateListOf
```

Comment:
Let's try to avoid using compose classes in ViewModels. It kind of locks us in to using compose if we want to write unit tests against the ViewModels themselves, which shouldn't strictly be necessary.